tests/foreman/api/test_product.py

```diff
@@ -122,3 +123,42 @@ def test_positive_update_1(self, attrs):
         for name, value in attrs.items():
             self.assertIn(name, new_attrs.keys())
             self.assertEqual(value, new_attrs[name])
+
+
+@run_only_on('sat')
+class RepositorySetsTestCase(APITestCase):
+    """Tests for ``katello/api/v2/products/<product_id>/repository_sets``."""
+
+    def test_repositoryset_enable(self):
+        """@Test: Enable repo from reposet by reposet, org and product
+
+        @Feature: Repository-set
+
+        @Assert: Repository was enabled
+
+        """
+        org = entities.Organization().create()
+        manifest_path = manifests.clone()
+        org.upload_manifest(path=manifest_path)
```

Comment:
`manifests_path` is a single-use variable. It should be possible to shrink this down:

```python
org = entities.Organization().create()
org.upload_manifest(path=manifests.clone())
```